[CIE-5504] c8yedge config docs (#4146)
SoftwareAG/c8y-docs: content/edge-kubernetes/k8-edge-connecting-edge-to-cloud-bundle/k8-edge-register-edge-on-cloud.md

layout: redirect
---

To remotely manage, configure, and monitor Edge through a {{< product-c8y-iot >}} cloud tenant, you must first register it as a device within the cloud tenant. This registration process requires providing the {{< product-c8y-iot >}} cloud tenant URI, along with an optional TLS/SSL key and certificate chain. These credentials authenticate Edge when connecting to the cloud via the MQTT protocol using X.509 certificate-based authentication. 

> If you installed Edge using the c8yedge tool, you can configure the cloud tenant URI, TLS/SSL key, and certificate chain using the command below.

Reviewer: Usual style is that we bold the tool name the first time it's mentioned in a section (outside of code blocks)
```suggestion
If you installed Edge using the **c8yedge** tool, you can configure the cloud tenant URI, TLS/SSL key, and certificate chain using the command below.
```

layout: redirect
---

> To remotely manage, configure, and monitor Edge through a {{< product-c8y-iot >}} cloud tenant, you must first register it as a device within the cloud tenant. This registration process requires providing the {{< product-c8y-iot >}} cloud tenant URI, along with an optional TLS/SSL key and certificate chain. These credentials authenticate Edge when connecting to the cloud via the MQTT protocol using X.509 certificate-based authentication. You can configure the cloud tenant URI, TLS/SSL key, and certificate chain using the `spec.cloudTenant` field in the Edge CR. For more details, refer to [Edge custom resource - Cloud Tenant](/edge-kubernetes/edge-custom-resource-definition/#cloudTenant). You can also achieve this using `c8yedge config --set cloudTenant.domain=<CLOUD-TENANT-URI> --set-file cloudTenant.tlsSecret.tls.key=<path/to/tls.key> cloudTenant.tlsSecret.tls.crt=<path/to/tls.crt>` command.

Reviewer 2: ```suggestion
To remotely manage, configure, and monitor Edge through a {{< product-c8y-iot >}} cloud tenant, you must first register it as a device within the cloud tenant. This registration process requires providing the {{< product-c8y-iot >}} cloud tenant URI, along with an optional TLS/SSL key and certificate chain. These credentials authenticate Edge when connecting to the cloud via the MQTT protocol using X.509 certificate-based authentication. You can configure the cloud tenant URI, TLS/SSL key, and certificate chain using the `spec.cloudTenant` field in the Edge CR. For more details, refer to [Edge custom resource > Cloud tenant](/edge-kubernetes/edge-custom-resource-definition/#cloudTenant). You can also achieve this using the `c8yedge config --set cloudTenant.domain=<CLOUD-TENANT-URI> --set-file cloudTenant.tlsSecret.tls.key=<path/to/tls.key> cloudTenant.tlsSecret.tls.crt=<path/to/tls.crt>` command.
```

> For general guidance on configuring Edge, see the [Modify Edge](/edge-kubernetes/manage-edge/#modify-edge) section.

Reviewer 2: ```suggestion
For general guidance on configuring Edge, see [Modifying Edge](/edge-kubernetes/manage-edge/#modify-edge.
```
